acceptance/tests/ssl/autosign_command.rb

```diff
@@ -32,9 +32,22 @@ def assert_key_generated(name)
   fqdn = master.execute('facter fqdn')
 
   step "Step 1: ensure autosign command can approve CSRs" do
+
+    # Our script needs to consume stdin (SERVER-1094)
+    # We should revert back to /bin/true once resolved
+    autosign_true_script = <<-EOF
+!#/bin/bash
+less
```

Comment:
Will this block if `PAGER` is defined? Is there another way to drain stdin without involving terminals and possibly hanging, something like:

```bash
#!/bin/bash
while read line
do
    : # noop command
done
exit 0
```